### PR TITLE
Search

### PR DESCRIPTION
**Pull request by ViktoriyaSynytsya**

### CODE REVIEW

File: specs/search_spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+describe('Protractor Demo App Search', function () {
+    it('Search', async function () {
+        await indexPage.open();
+        await indexPage.waitForPageLoaded();
+        await search_page.search('dress'); 
+        //await search_page.waitForPageLoaded();
+        expect(await search_page.getCountOfResults()).toEqual('7 results have been found.');
+        await search_page.getFirstResult();
+        expect(await search_page.getNameOfFirstProduct()).toEqual('Printed Summer Dress');
```
Comment (nbabii) [inline specs/search_spec.js]: `getNameOfFirstProduct()` method is not related to the search page, it should be in a different page object.

File: specs/search_spec.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+let search_page = require("../page_objects/search_page");
+
+describe('Protractor Demo App Search', function () {
+    it('Search', async function () {
+        await indexPage.open();
+        await indexPage.waitForPageLoaded();
+        await search_page.search('dress'); 
+        //await search_page.waitForPageLoaded();
+        expect(await search_page.getCountOfResults()).toEqual('7 results have been found.');
+        await search_page.getFirstResult();
```
Comment (nbabii) [inline specs/search_spec.js]: `getFirstResult()` please rename to `clickFirstResult()` or  `openFirstResult()`

**Review by nbabii — CHANGES_REQUESTED** [review]

File: specs/search_spec.js (hunk 1 above)
Comment (ViktoriyaSynytsya) [reply specs/search_spec.js]: Fixed

File: specs/search_spec.js (hunk 2 above)
Comment (ViktoriyaSynytsya) [reply specs/search_spec.js]: Fixed

**Pull request merged**